# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/reftests/svg/filters/css-filter-chains/reftest.list
# Commit: b7f88e5c537b
# Full Hash: b7f88e5c537bb7d64ba9d4cb58b052650d510638
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-02-08 17:32:27
# Regressor Bug: 1749380
# File Overlap Count: 1
# Description:
#   Bug 1749380 - Improve how WR handles bounding rects for off-screen surfaces r=gfx-reviewers,kvark
#   
#   This patch introduces a number of subtle but important changes
#   to how we deal with off-screen surfaces. The overall goals are:
#   
# ==============================================================================

diff -r 8e9c1ffdeed8 -r b7f88e5c537b layout/reftests/svg/filters/css-filter-chains/reftest.list
--- a/layout/reftests/svg/filters/css-filter-chains/reftest.list	Mon Feb 07 22:02:35 2022 +0000
+++ b/layout/reftests/svg/filters/css-filter-chains/reftest.list	Mon Feb 07 22:14:47 2022 +0000
@@ -2,6 +2,6 @@
 # e.g. filter: blur(3px) grayscale(0.5) invert(0.2);
 
 # Some platforms render this complex filter chain a little differently, and that's ok.
-fuzzy(4-6,12000-19484) fuzzy-if(swgl,5-10,13600-20088) == long-chain.html long-chain-ref.html # Win10: Bug 1258241
+fuzzy(4-6,12000-19520) fuzzy-if(swgl,5-10,13600-20260) == long-chain.html long-chain-ref.html # Win10: Bug 1258241
 == moz-element.html moz-element-ref.html
 fuzzy-if(!useDrawSnapshot,13-15,7670-7982) fuzzy-if(!useDrawSnapshot&&swgl,11-12,14052-14056) == same-filter.html same-filter-ref.html